microsoft-365/security/defender-endpoint/configure-server-endpoints.md

```diff
@@ -117,7 +117,7 @@ For more information, see [Integration with Azure Defender](azure-server-integra
 ### Prerequisites
 
 **Prerequisites for Windows Server 2012 R2** 
-If you have fully updated your machines with the latest [monthly rollup](/troubleshoot/windows-client/deployment/standard-terminology-software-updates.md#monthly-rollup) package, there are **no** additional prerequisites.
+If you have fully updated your machines with the latest [monthly rollup](/troubleshoot/windows-client/deployment/standard-terminology-software-updates.md#monthly-rollup) package, there are **no** additional prerequisites. Make sure that System Center Endpoint Protection is not installed.
```

Comment:
```suggestion

If you have fully updated your machines with the latest [monthly rollup](/troubleshoot/windows-client/deployment/standard-terminology-software-updates.md#monthly-rollup) package, there are **no** additional prerequisites. However, but make sure that System Center Endpoint Protection is not installed.
```

Reply:
I think removing "but" word makes the sentence clearer:

"However, make sure that System Center Endpoint Protection is not installed."

Main thing is to point out not use SCEP! :-)

Reply:
Fully agree with you, my mistake. Thanks for pointing it. Have changed the suggestion accordingly. Could you please commit now? Thanks so much.